src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -6,12 +6,20 @@ export function focusElement(selectorOrElement, options) {
     typeof selectorOrElement === 'string'
       ? document.querySelector(selectorOrElement)
       : selectorOrElement;
+  let prevTabIdx = null;
 
   if (el) {
-    if (el.tabIndex <= 0) {
+    prevTabIdx = el.getAttribute('tabindex');
+    if (prevTabIdx && prevTabIdx <= 0) {
```

Comment:
I know this is basically existing code but I'm wondering what the point of this check is. Set the element's tabindex to -1 if it's already -1?

Reply:
I think it's related to IE11, but I don't remember exactly.